src/plugins/data/public/ui/typeahead/_suggestion.scss

```diff
@@ -10,12 +10,13 @@ $kbnTypeaheadTypes: (
   position: relative;
 
   .kbnTypeahead__popover {
+    max-height: 20vh;
```

Comment:
it looks like extra style. I see you set the 'max-height' as inline styles for that element
![image](https://user-images.githubusercontent.com/20072247/88545018-113f1100-d023-11ea-8dc9-754b72dd3e9b.png)